docs/known-issues.md

```diff
@@ -29,6 +29,12 @@ Symbolic links to directories within the *originals* folder are supported if the
 
 It is also not supported to mount a symbolic link as *storage* folder, or to use links inside the *storage* folder.
 
+### Failed Imports with Network Storage and Docker
+
+When network-based storage (e.g. SMB/CIFS) is combined with running PhotoPrism under Docker, the originals and import directories should be mounted to the container as a single mount point, rather than as two separate mounts. If this is not done, you may observe 'empty file' errors when the import runs, due to caching of zero-length inode data after the files are moved into the library.
```

Comment:
Could you please provide an example of how to do it? Explaining a caveat is good, providing the solution directly is better.

Reply:
Also note that we have made changes in the development preview that might prevent this. These still need to be tested. However, we do not have a test environment for this particular case.